model/customlist.py

```diff
@@ -61,9 +61,18 @@ class CustomList(Base):
     # interface for managing this.
 
     def __repr__(self):
+        # TODO PYTHON3 repr is unicode string
         return (u'<Custom List name="%s" foreign_identifier="%s" [%d entries]>' % (
             self.name, self.foreign_identifier, len(self.entries))).encode('utf8')
 
+    def __lt__(self, other):
```

Comment:
Why only this method?

Reply:
This one makes `sorted` work if you run it on a homogenous list of `CustomList` objects.

Reply:
Can we rely on `sorted` only using this operator? It seems like an internal Python implementation detail. 

I also found https://docs.python.org/3.3/library/functools.html#functools.total_ordering when I was trying to understand this.

Reply:
That's a good find, I made `Identifier` and `CustomList` into @total_orderings.